### ISSUE TITLE
[TTT] Loja Rápida

### ISSUE THREAD

**Issue by hcjordao** [quasemago/zrageservers#168]
Ao apertar uma tecla a ser definida 2 vezes bem rapido abrir um menu configuravel dos itens mais rapidos a serem comprados na loja.

Funcionalidade:
- Menu tem um limite de X items a serem configurados
- Menu seria mostrado com o numero de itens e na ultima opção com um configurar itens. Ao apertar o item de configurar, abriria uma lista de itens e ao selecionar tu selecionaria qual posição desse menu tu quer inserir os itens.
- Acho que 3 itens ta bom de começo. Aposto que todos irão amar essa opção.

**Comment by Tardigrada-TTT** [comment]
Concordo, mesmo já tendo as binds, isso irá faciltiar, principalmente aos novatos
